class UnexpectedResponse(RuntimeError):
    pass


class NetworkOutage(RuntimeError):
    pass
